physical/foundationdb/fdb-go-install.sh

```diff
@@ -0,0 +1,330 @@
+#!/bin/bash -eu
+#
+# fdb-go-install.sh
+#
+# Installs the FoundationDB Go bindings for a client. This will download
+# the repository from the remote repo either into the go directory
+# with the appropriate semantic version. It will then build a few
+# generated files that need to be present for the go build to work.
+# At the end, it has some advice for flags to modify within your
+# go environment so that other packages may successfully use this
+# library.
+#
+
+DESTDIR="${DESTDIR:-}"
+FDBVER="${FDBVER:-5.1.0}"
```

Comment:
Should this be something like a latest tag? I just wonder whether this will end up just being out of date. Future PRs could bump it, so if it's a good idea to use "known good versions" that's a good argument, but just trying to understand the situation.

Reply:
Yes, this is a known-good version kind of situation. The script is taken from the 5.1.x branch, so it defaults to that. FDBVER can be set when calling the script to get another version, and it should work just fine.

Having the script in the repo made obvious sense when it was called from bootstrap, but since that isn't the case anymore, I guess we can remove it and update the documentation to match. Let me know which way you are leaning.

Reply:
I'm happy for the script to be in the repo, just want to understand these questions for the inevitable user queries :-)